content/components/dialog.mdx

```diff
@@ -0,0 +1,265 @@
+
+The header region provides a slot for custom content in place of the default title/description, though a title is still required and may be visually hidden. Secondary action icon buttons may be placed next to the close button.
+
+<Box display="grid" gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={5}>
+  <div>
+    <img src="https://user-images.githubusercontent.com/293280/175849372-a9d33d3b-96f3-497e-bad6-70df4d91072a.png" alt="" />
+    <Caption>Header with the default title variant.</Caption>
+  </div>
+  <div>
+    <img src="https://user-images.githubusercontent.com/293280/175849374-6a71d8a0-da61-4745-a059-4560d053089b.png" alt="" />
+    <Caption>Header with large title variant.</Caption>
+  </div>
+</Box>
+
+<Box display="grid" gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={5}>
+  <div>
+    <img width="464" alt="" src="https://user-images.githubusercontent.com/18661030/185478040-be5a3552-9ed1-40ef-a713-3d6ac8f6b697.png" />
+    <Caption>Header with custom variant and a secondary action button.</Caption>
+  </div>
+  <div>
+    <img width="464" alt="" src="https://user-images.githubusercontent.com/18661030/185478144-87887a98-e541-4a23-bd1c-6cb174a453a7.png"/>
+    <Caption>Command palette with a visually hidden title.</Caption>
+  </div>
+</Box>
+
+### Subheader region
+
+The subheader region is an optional space for interactive controls. Use it to display a search field, a filter menu, or a local navigation component.
+
+<Box display="grid" gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={5}>
+  <div>
+    <img src="https://user-images.githubusercontent.com/293280/175849377-ef70f4e7-2978-415c-9da1-a2ba2f33bdc3.png" alt="" />
+    <Caption>Header displaying a subheader with a search field.</Caption>
+  </div>
+  <div>
+    <img src="https://user-images.githubusercontent.com/293280/175849376-ef25a1ba-b5b7-4ae0-8d9a-78095932886a.png" alt="" />
+    <Caption>Header displaying a subheader with a local navigation.</Caption>
+  </div>
+</Box>
+
+### Body region
+
+The body region is the main content of the Dialog. It can contain any content that is relevant to the task at hand. The body region is scrollable if the content exceeds the height of the Dialog.
+
+<Box display="grid" gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={5}>
+  <div>
+    <img width="650" alt="" src="https://user-images.githubusercontent.com/18661030/185506131-1436b06b-a8fb-4de8-b9b2-4715ab0a63d2.png"/>
+    <Caption>Body content will scroll vertically if the content exceeds the overall Dialog height.</Caption>
+  </div>
+  <div>
+    <img width="650" alt="" src="https://user-images.githubusercontent.com/18661030/185478320-2a4a3053-dca3-4048-b7c4-d5a39a83acd6.png" />
+    <Caption>The body may contain additional content such as banners and form controls.</Caption>
```

Comment:
I think we can simplify this example a bit:

Before:

> ![image](https://user-images.githubusercontent.com/293280/185690249-94e18546-9275-43e3-a5a7-28783a1c787f.png)

After:

> ![image](https://user-images.githubusercontent.com/293280/185690317-526f4baf-4cfc-40ca-8587-9a0c64d5bd6a.png)

```suggestion
    <img width="650" alt="" src="https://user-images.githubusercontent.com/293280/185690317-526f4baf-4cfc-40ca-8587-9a0c64d5bd6a.png" />
    <Caption>The body may contain additional content such as banners and form controls.</Caption>
```

Reply:
Thanks @vdepizzol! I updated the image to be closer to your recommendation. The height/width combo in the one you shared is in our disallowed list and I want this to be a little more accurate if possible!